test/src/test/java/hudson/console/HyperlinkNoteTest.java

```diff
@@ -72,6 +80,28 @@ public void textWithNewlinesModelHyperlinkNote() throws Exception {
                 containsString(new ModelHyperlinkNote("", 0).extraAttributes()),
                 containsString(">" + noteTextSanitized + "</a>")));
     }
+    
+    @Test
+    public void textWithSingleQuote() throws Exception {
+    	FreeStyleProject upstream = r.createFreeStyleProject("upstream");
+    	r.createFreeStyleProject("d0wnstr3'am");
+    	upstream.getPublishersList().add(new BuildTrigger("d0wnstr3'am", Result.SUCCESS));
+    	FreeStyleBuild b = r.buildAndAssertSuccess(upstream);
+        r.waitUntilNoActivity();
```

Comment:
Please use consistent indentation when adding new code.  The changing indentation within this single method makes the pull request more difficult to read.

Most integrated development environments (Netbeans, IntelliJ, Visual Studio Code) provide automatic formatting that can be applied to subsections (like this new block of code).
```suggestion
    r.waitUntilNoActivity();
```